src/Lykke.Job.BackgroundWorker/Modules/JobModule.cs

```diff
@@ -48,6 +61,12 @@ protected override void Load(ContainerBuilder builder)
                 .SingleInstance()
                 .WithParameter(TypedParameter.From(TimeSpan.FromSeconds(30)));
```

Comment:
Magic number

Reply:
Yes, I agree with you. I'll remove this typed parameter because I didn't found appropriate constructor which accept one argument of TimeSpan type.